### ISSUE TITLE
aria-labelledby property got swallowed by RadioGroup control

### ISSUE THREAD

**Issue by jackwang72** [carbon-design-system/carbon#7333]
## Environment

> Operating system

> Browser

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue
Created a toolkit control as follows:
`**```
<RadioGroup
direction='vertical'
options={options}
aria-labelledby='dtScopeRadioLabel'
checked={this.state.checked} />
```**`
Observed that aria-labelledby was swallowed after the react component is mounted, which brakes a11y

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by dakahn** [comment]
@jackwang72 can you recreate your problem in the codesandbox in the template?

**Comment by tw15egan** [comment]
@jackwang72 I'm not seeing any a11y violations without the `aria-labelledby` attribute so I'm going to close this

<img width="1119" alt="Screen Shot 2021-02-04 at 12 20 16 PM" src="https://user-images.githubusercontent.com/11928039/106950504-5a170700-66e3-11eb-8d1a-fb0a5a2e9876.png">

Feel free to reopen if you can provide us an example recreating the issue you are facing. Thanks!
